diff --git a/node_modules/chokidar/lib/fsevents-handler.js b/node_modules/chokidar/lib/fsevents-handler.js
index 007a7ec..aad0883 100644
--- a/node_modules/chokidar/lib/fsevents-handler.js
+++ b/node_modules/chokidar/lib/fsevents-handler.js
@@ -5,11 +5,7 @@ const sysPath = require('path');
 const { promisify } = require('util');
 
 let fsevents;
-try {
-  fsevents = require('fsevents');
-} catch (error) {
-  if (process.env.CHOKIDAR_PRINT_FSEVENTS_REQUIRE_ERROR) console.error(error);
-}
+if (process.env.CHOKIDAR_PRINT_FSEVENTS_REQUIRE_ERROR) console.error(error);
 
 if (fsevents) {
   // TODO: real check
