diff --git a/node_modules/sass/sass.dart.js b/node_modules/sass/sass.dart.js
index 8d6af25..3730151 100644
--- a/node_modules/sass/sass.dart.js
+++ b/node_modules/sass/sass.dart.js
@@ -3,7 +3,7 @@
 
 var dartNodePreambleSelf = typeof global !== "undefined" ? global : window;
 
-var self = Object.create(dartNodePreambleSelf);
+var self = dartNodePreambleSelf;
 
 self.scheduleImmediate = self.setImmediate
     ? function (cb) {
